pkg/operator/resourcesynccontroller/resourcesync_controller.go

```diff
@@ -135,10 +135,13 @@ func (c *ResourceSyncController) sync() error {
 	}
 
 	switch operatorSpec.ManagementState {
+	case operatorv1.Managed:
 	case operatorv1.Unmanaged:
 		return nil
 	case operatorv1.Removed:
-		// TODO: Should we try to actively remove the resources created by this controller here?
+		// TODO probably just fail
+		return nil
+	default:
```

Comment:
no warning?

Reply:
Warning is here https://github.com/openshift/cluster-openshift-apiserver-operator/pull/147

I don't want to spam events/logs (it will create same warning for every controller loop ;-)